test/src/Makefile

```diff
@@ -110,6 +110,11 @@ $(ALL_DIRS):
 %/hellobundle.so: $(LIBHELLO_SRC) %
 	$(CC) $(ARCH_FLAGS) -bundle $< -o $@
 
+# Run post-make builds (inconsistent binaries)
+.PHONY: post-make
+post-make:
```

Comment:
Best practice would be to declare the dependencies of this make target nonetheless, i.e.:

```makefile
post-make: $(USE_TARGETS)
```

This way, a `make post-make` (maybe name the target `inconsistent` and the shell script `make-inconsisten.sh`?) will simply work instead of terminating as follows (though I give you credit for catching this in the shell script):

```console
$ make post-make
Makefile:49: USE - Option either unset or invalid. Using a safe fallback.
Makefile:50: USE - Valid choices: all, 10.6-xcode3.2.6, 10.11-xcode7.3.
bash post-make.sh i386 x86_64 fat-i386-x86_64
Error: Missing file: i386/libhello.dylib. Did you run make?
make: *** [post-make] Error 1
```